static/styles/portico/navbar_new_style.css

```diff
@@ -0,0 +1,575 @@
+.top-menu {
+    position: fixed;
+    width: 100%;
+    background: rgba(0, 0, 50, 0.4);
+    backdrop-filter: blur(10px);
+    backdrop-filter: blur(10px);
+    font-family: var(--font-ss3);
+    color: white;
+    z-index: 10;
+    overflow: hidden;
+}
```

Comment:
This can probably be called just `navbar.css` -- it seems nice to use the cleaner names for the post-redesign version. (I regret having a `new-style` CSS class that's still present in the web app 5 years after being introduced).

Reply:
Done.